SpywareFilter/sections/specific.txt

```diff
@@ -41,6 +41,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||spush.toptoon.com/log/counter.jpeg?
```

Comment:
I think you can add this `||spush.toptoon.com/log/`.

Reply:
Resolved. https://github.com/AdguardTeam/AdguardFilters/pull/98854/commits/e13b7915260332ec11794750da9fc7775c229b95